index.html

```diff
@@ -12,14 +12,42 @@
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
+
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
 </head>
 
 <body>
 
   
-     
+
+    <div class="nav-head">
+        
+        <div class="fun-bus">  
+          <p>Fun Bus</p>  
```

Comment:
you could probably have gotten the same effect easier by making this an h1 instead of a p inside a div